edk2toolext/invocables/edk2_setup.py

```diff
@@ -48,6 +48,49 @@ def GetLoggingLevel(self, loggerType):
         '''
         pass
 
+    # ####################################################################################### #
+    #                           Supported Values and Defaults                                 #
+    # ####################################################################################### #
+    def GetPackagesSupported(self):
```

Comment:
I think it could be worthwhile to have base settings classes where we include these functions like GetArchitecturesSupported

Reply:
can't really because these command line options are not defined except in the final instance